oVirt populator build fails when done in a git workspace

The oVirt Populator fails when done in a git workspace with the following error:
```
make build-ovirt-populator-image
go fmt ./pkg/... ./cmd/...
pkg/apis/forklift/v1beta1/plan.go
pkg/controller/plan/migration.go
pkg/forklift-api/webhooks/validating-webhook/admitters/provider-admitter.go
go mod tidy
go vet ./pkg/... ./cmd/...
bazel run //:gazelle
INFO: Analyzed target //:gazelle (1 packages loaded, 2 targets configured).
INFO: Found 1 target...
Target //:gazelle up-to-date:
  bazel-bin/gazelle-runner.bash
  bazel-bin/gazelle
INFO: Elapsed time: 0.160s, Critical Path: 0.01s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Build completed successfully, 1 total action
/usr/bin/podman build -f hack/ovirt-populator/Containerfile -t default-route-openshift-image-registry.apps.ocp4.example.com/konveyor-forklift/ovirt-populator:latest .
[1/2] STEP 1/4: FROM registry.access.redhat.com/ubi8/go-toolset:1.18.9-8 AS builder
[1/2] STEP 2/4: ENV GOPATH=$APP_ROOT
--> Using cache 4bb55f78d7e8f54c427dfeb64cf3dc6c247a80e931e31fc1c6fa5225715b3c9d
--> 4bb55f78d7e8
[1/2] STEP 3/4: COPY . .
--> bf08650bca66
[1/2] STEP 4/4: RUN CGO_ENABLED=0 go build -o ovirt-populator github.com/konveyor/forklift-controller/cmd/ovirt-populator
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
Error: building at STEP "RUN CGO_ENABLED=0 go build -o ovirt-populator github.com/konveyor/forklift-controller/cmd/ovirt-populator": while running runtime: exit status 1
make: *** [Makefile:342: build-ovirt-populator-image] Error 1
```

--- a/hack/ovirt-populator/Containerfile
+++ b/hack/ovirt-populator/Containerfile
@@ -1,10 +1,14 @@
 FROM registry.access.redhat.com/ubi8/go-toolset:1.18.9-8 as builder
 ENV GOPATH=$APP_ROOT
-COPY . .
 
+COPY go.mod ./
+COPY go.sum ./
+COPY cmd/ovirt-populator/ovirt-populator.go ./
+COPY vendor/ ./vendor/
 # When debugging by replacing the lib-volume-populator with a local modified copy
 # COPY . ./
-RUN CGO_ENABLED=0 go build -o ovirt-populator github.com/konveyor/forklift-controller/cmd/ovirt-populator
+
+RUN CGO_ENABLED=0 go build -o ovirt-populator
 
 FROM quay.io/centos/centos:stream9-minimal
 COPY --from=builder /opt/app-root/src/ovirt-populator /usr/local/bin

